### PR TITLE
[4.0] Show hide strings

### PR DESCRIPTION
**Pull request by brianteeman**
We have a complete mix of using the word show in our option strings
For example we have
Show Option - yes/no
Show Option - show/hide
Option - show/hide

This is a first pass at making them consistently 
Option - show/hide

To test/review its easiest to select a components options and then use the sidebar to move between components

If approved then I will do further PR to address this inconsistency in the UI everywhere

### CODE REVIEW

**Comment by ghost** [comment]
I have tested this item :white_check_mark: successfully on dd6cffb5a568b98ea0b07a3057f4a9615c134df7<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/19242">issues.joomla.org/tracker/joomla-cms/19242</a>.</sub>

**Comment by wilsonge** [comment]
I think this makes lots of sense

**Comment by brianteeman** [comment]
@wilsonge should i add the rest to this PR or in another PR?

**Comment by wilsonge** [comment]
If you're ok with it add the rest to this PR and ping me when you're done and I'll get it merged

**Comment by brianteeman** [comment]
ok

**Comment by brianteeman** [comment]
@wilsonge I finished with this

File: administrator/components/com_menus/forms/item_component.xml [hunk 1]
```diff
@@ -70,8 +70,8 @@
 				default=""
 				useglobal="true"
 				>
-				<option value="1">JYES</option>
-				<option value="0">JNO</option>
+				<option value="1">JSHOW</option>
```
Comment (Quy) [inline administrator/components/com_menus/forms/item_component.xml:73]: The majority are `Hide/Show`. This should be in the same order to be consistent.

Comment (brianteeman) [reply administrator/components/com_menus/forms/item_component.xml:73]: I assumed that there was a specific reason for this one to be reversed so i did not change the order

File: modules/mod_login/mod_login.xml [hunk 2]
```diff
@@ -96,8 +96,8 @@
 					class="switcher"
 					default="0"
 					>
-					<option value="1">JYES</option>
-					<option value="0">JNO</option>
+					<option value="1">JSHOW</option>
```
Comment (Quy) [inline modules/mod_login/mod_login.xml:99]: This one should definitely be swapped. `Greeting` is green when `Yes`. `Profile Link` is green when `No`. I find it confusing.

Comment (brianteeman) [reply modules/mod_login/mod_login.xml:99]: Its really beyond the scope of this PR

File: administrator/components/com_menus/forms/item_component.xml (hunk 1 above)
Comment (wilsonge) [reply administrator/components/com_menus/forms/item_component.xml:73]: @C-Lodder given you're the one that reversed the orders

Comment (C-Lodder) [reply administrator/components/com_menus/forms/item_component.xml:73]: It's out of scope but it you want to reorder them in this PR, feel free.

Comment (C-Lodder) [reply administrator/components/com_menus/forms/item_component.xml:73]: Please be sure to create an issue incase it doesnt go into this PR so we dont forget

**Comment by wilsonge** [comment]
@Quy do you want to do a PR addressing the consistency issues you raised :)

**Comment by brianteeman** [comment]
Thanks kindly

**Comment by wilsonge** [comment]
Thanks!

**Pull request merged**